bugbug/model.py

```diff
@@ -73,14 +73,22 @@ def bugs():
             explainer = shap.TreeExplainer(self.clf)
             shap_values = explainer.shap_values(X_train)
 
-            print('Feature ranking (top 20 features):')
             # Calculate the values that represent the fraction of the model output variability attributable
             # to each feature across the whole dataset.
             shap_sums = np.abs(shap_values).sum(0)
             rel_shap_sums = shap_sums / shap_sums.sum()
-            indices = np.argsort(rel_shap_sums)[::-1][:20]
-            for i, index in enumerate(indices):
-                print('{}. \'{}\' ({})'.format(i + 1, feature_names[index], rel_shap_sums[index]))
+
+            cut_off_percentage = 15
+            cut_off_value = (cut_off_percentage / 100) * np.amax(rel_shap_sums)
+
+            top_feature_indices = np.where(rel_shap_sums >= cut_off_value)[0][::-1]
+            top_vals = np.take(rel_shap_sums, top_feature_indices)
+            top_vals = np.column_stack((top_vals, top_feature_indices))
+            top_vals = top_vals[top_vals[:, 0].argsort()][::-1]
+
+            print('Feature ranking (atleast {}% of the top feature):'.format(cut_off_percentage))
```

Comment:
Let's print the number of top features, instead of the fact that it's XX% of the top.